### ISSUE TITLE
ESLint port

### ISSUE THREAD

**Issue by natemoo-re** [natemoo-re/tslint-stencil#9]
As TSLint is deprecated, it would be great to support an ESLint port of `tslint-stencil`

**Comment by vidarc** [comment]
Maybe you could work with the stencil guys?

In case you haven't seen: https://github.com/ionic-team/stencil-eslint/issues/1

Love your rules, so would be great to have an easy transition over.

**Comment by natemoo-re** [comment]
Yep, they've reached out about it! I'm hoping to port these rules to that official package and deprecate this one.

Thanks for linking the issues for future readers. 😄
